web/src/containers/Admin/SetupWizard/index.css

```diff
@@ -158,27 +158,27 @@
     margin: auto;
     margin-top: 7px;
 }
-.sell {
+.wizard-container .sell {
```

Comment:
These classes are overriding some of the styles on the pro-trade page. I think it is better to wrap all of the setup wizard classes with the wizard-container class, for now, to make sure this will not happen in other cases. @abeikverdi